.travis/apisix_cli_test.sh

```diff
@@ -391,6 +391,52 @@ fi
 
 git checkout conf/config.yaml
```

Comment:
We don't need to checkout in the middle of a group of test, as we will change it later.